src/webserver/Listener.h

```diff
@@ -141,7 +141,7 @@ class Detector : public std::enable_shared_from_this<Detector<PlainSession, SslS
 };
 
 void
-make_websocket_session(
+make_WebsocketSession(
```

Comment:
Why not makeWebsocketSession ?

Reply:
Same like we have other `make_*` functions, don't want to change that as well in this pr - maybe another time :)